plugin.php

```diff
@@ -318,6 +318,12 @@ function rest_validate_request_arg( $value, $request, $param ) {
 			}
 		}
 
+		if ( isset( $args['minimum'] ) && isset( $args['maximum'] ) ) {
```

Comment:
> It feels like we are just inventing this adhoc?

Huh? `minimum` and `maximum` are a part of the spec:

* https://github.com/json-schema/json-schema/wiki/Minimum-and-exclusiveminimum
* https://github.com/json-schema/json-schema/wiki/Maximum-and-exclusivemaximum

Although, I'll include a type check for `integer` and `number`